document-generator-core/src/main/resources/avro/document-generation-completed.avsc

```diff
@@ -0,0 +1,48 @@
+{
+  "type": "record",
```

Comment:
The schemas shouldn't be copied here. They are duplicated to the schemas in chs-kafka-schemas repo. https://github.com/companieshouse/kafka-models holds the generated models to allow re-use by the producer and consumers. The kafka-models repo has chs-kafka-schemas as a submodule to prevent needing to duplicate the schemas.

Reply:
Agreed that this will be covered by the next change to keep the changes in each PR small.